apps/preview/nuxt/pages/showcases/Card/CardDefault.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <div class="flex flex-wrap gap-4 lg:gap-6 lg:flex-nowrap">
+    <div
+      v-for="({ image, title, description, button }, index) in cardDetails"
+      :key="`${title}-${index}`"
+      class="flex flex-col min-w-[325px] max-w-[375px] lg:w-[496px] relative border border-neutral-200 rounded-md hover:shadow-xl"
+    >
+      <a
+        class="absolute w-full h-full z-1 focus-visible:outline focus-visible:outline-offset focus-visible:rounded-md"
+        href="/"
+      />
+      <img :src="image" alt="Card alt" class="object-cover h-auto rounded-t-md aspect-video" />
```

Comment:
What do you think about switching `alt` value to `title` 
```suggestion
      <img :src="image" :alt="title" class="object-cover h-auto rounded-t-md aspect-video" />
```

Reply:
ok